### DISCUSSION TITLE
PubLayNet 数据集的json标签是直接可以用的吗？不需要转换格式？

### DISCUSSION THREAD

**Discussion by nissansz** [PaddlePaddle/PaddleOCR#12215] [Q&A] [answered]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

PubLayNet 数据集的json标签是直接可以用的吗？不需要转换格式？

 https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/ppstructure/layout/README_ch.md

**Comment by SWHL** [comment, marked as answer]
是的呢
